build_runner_core/lib/src/generate/build_impl.dart

```diff
@@ -855,3 +859,15 @@ String _actionLoggerName(
       : primaryInput.uri;
   return '${phase.builderLabel} on $asset';
 }
+
+String _twoDigits(int n) {
```

Comment:
Are we re-implementing `DateFormatter` to avoid a dependency on `intl`?

How about using `padLeft` for this?

String _twoDigits(int n) => '$n'.padLeft(2, '0');

Reply:
> Are we re-implementing `DateFormatter` to avoid a dependency on `intl`

Yes, it seems crazy basic date formatting doesn't come with DateTime but :man_shrugging: 


> How about using `padLeft` for this?

Sure, I just copied the internal impl from DateTime but I can change it. I will also remove `_fourDigits` since I realized its totally unecessary for this case (we don't have to deal with old dates).